### PR TITLE
memory: don't use 2 images for double buffer

### PR DESCRIPTION
**Pull request by pelikhan**
- Removed extra screen image
- adding profiling calls

### CODE REVIEW

File: options.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 namespace microcode {
     export class Options {
-        public static fps: boolean = false
+        public static fps: boolean = true
+        public static profiling = true
```
Comment (mmoskal) [inline options.ts:4]: turn off by default?

**Review by mmoskal — COMMENTED** [review]
lgtm

File: options.ts (hunk 1 above)
Comment (pelikhan) [reply options.ts:4]: yes mistake

**Comment by pelikhan** [comment]
Tested on HW lookin good.

**Pull request merged**